src/linter/linter.ts

```diff
@@ -372,6 +372,33 @@ function checkUnmatchedPatterns(patterns: FilePattern[], patternsMatch: Set<stri
 		return acc;
 	}, [] as FilePattern[]);
 
+	// Check if the Glob pattern is actually a file that exists, but the linter is not able to process.
+	if (unmatchedPatterns.length) {
+		const notProcessedFiles = new Set<string>();
+		const rootFileReader = createReader({
+			fsBasePath: options.rootDir,
+			virBasePath: "/",
+		});
+		// TODO: Define a better way to get all files. Exclude node_modules, .git, etc.
+		const allFiles = await rootFileReader.byGlob("/**/*");
+		const filePaths = allFiles.map((file) => file.getPath());
+
+		for (const pattern of unmatchedPatterns) {
+			if (filePaths.some((filePath) => filePath.includes(pattern))) {
```

Comment:
`pattern` could also be something like `"**/*.pdf"`

Reply:
Thanks!
Now we check files against the actual pattern